src/sentry/static/sentry/less/shared-components.less

```diff
@@ -155,7 +155,8 @@ dl.flat dd {
   margin-left: -1px;
 }
 
-.btn-group .btn + .dropdown {
+.btn-group .btn + .dropdown,
+.btn-group .btn + .guide-anchor .dropdown {
```

Comment:
nit: would prefer if this wasn't in shared components (not sure if there's an easy way though so ignore if not)

Reply:
what do you mean by shared components?

Reply:
I think `shared-components.less` is intended for shared, highly reusable stuff